src/Init/Core.lean

```diff
@@ -78,6 +99,13 @@ instance thunkCoe : CoeTail α (Thunk α) where
 abbrev Eq.ndrecOn.{u1, u2} {α : Sort u2} {a : α} {motive : α → Sort u1} {b : α} (h : a = b) (m : motive a) : motive b :=
   Eq.ndrec m h
 
+/-! # definitions  -/
+
+@[inherit_doc True.intro] def trivial : True := ⟨⟩
+
+-- proof irrelevance is built in
+theorem proofIrrel {a : Prop} (h₁ h₂ : a) : h₁ = h₂ := rfl
```

Comment:
This doesn't follow our usual naming convention for theorems

Reply:
This was on the list from #1897

Reply:
I made the proofIrrel since it seemed to require minimal changes.  I'll try to re-evaluate on a case-by-case basis before merging.